.github/workflows/lint-js-css.yml

```diff
@@ -0,0 +1,43 @@
+name: Lint JS, CSS
+
+on:
+  pull_request:
+    paths:
```

Comment:
Do we need to require paths explicitly? We already have specified paths in the `npm:lint*` commands.